src/validators/companyCheckName.validator.js

```diff
@@ -2,8 +2,9 @@
 
 const Joi = require('joi')
 const BaseValidator = require('./base.validator')
+const Application = require('../models/application.model')
 
-const COMPANY_NAME_MAX_LENGTH = 170
+const COMPANY_NAME_MAX_LENGTH = Application.tradingName.length.max
```

Comment:
I don't think you really need to use a new constant here, you could just include it below as has been done in the other validators.